packages/grid/backend/grid/api/syft/syft.py

```diff
@@ -28,6 +33,20 @@ async def syft_metadata() -> Response:
     )
 
 
+@router.delete("", response_model=str)
+async def delete(current_user: UserPrivate = Depends(get_current_user)) -> Response:
+    # If current user is the node owner ...
+    if current_user.role == node.roles.owner_role.id:
```

Comment:
This access the node (and the db) directly and I'm under the impression that we should communicate with the node via `get_client`. What is your take @IonesioJunior?

Reply:
I agree, let me create a client interface to accomplish this. :)

Reply:
I agree, let me create a client interface to accomplish this. :)